core/src/main/java/org/vertexium/query/DefaultGraphQueryIterableWithAggregations.java

```diff
@@ -55,7 +62,30 @@ public static boolean isAggregationSupported(Aggregation agg) {
             Map<String, AggregationResult> nestedResults = getNestedResults(agg.getNestedAggregations(), entry.getValue());
             buckets.add(new TermsBucket(key, count, nestedResults));
         }
-        return (TResult) new TermsResult(buckets);
+        return new TermsResult(buckets);
+    }
+
+    private HistogramResult getCalendarFieldHistogramResult(final CalendarFieldAggregation agg, Iterator<T> it) {
+        String propertyName = agg.getPropertyName();
+        final Calendar calendar = GregorianCalendar.getInstance(agg.getTimeZone());
+        Map<Integer, List<T>> elementsByProperty = getElementsByProperty(it, propertyName, new ValueConverter<Integer>() {
+            @Override
+            public Integer convert(Object o) {
+                Date d = (Date) o;
+                calendar.setTime(d);
+                //noinspection MagicConstant
```

Comment:
IntelliJ complains that there ISN'T a constant referenced on this line. It wants it to be a `Calendar` magic constant.